inference/src/backends/base.py

```
@@ -322,7 +343,9 @@ def _download_and_concatenate_reference(
         cumulative_samples = 0
 
         for i, url in enumerate(urls):
-            logger.info(f"[{self.name}] Downloading reference audio {i + 1}/{len(urls)} from {url[:80]}...")
+            logger.info(
+                f"[{self.name}] Downloading reference audio {i + 1}/{len(urls)} from {url[:80]}..."
```

Ruff (0.15.12)

[warning] Logging statement uses f-string

(G004)